modules/fedimint-dummy-tests/tests/tests.rs

```diff
@@ -0,0 +1,34 @@
+use fedimint_core::{sats, Amount};
```

Comment:
Does this mean that the fixtures now work with external modules to some extent?

Reply:
Yep this is the first passing test written in the modular test framework :smile: 

Over the next couple weeks I'll be converting our old tests to use the new framework to test the new clients.

Reply:
Amazing